src/components/play-bar/PlayButton.js

```diff
@@ -137,17 +142,28 @@ class PlayButton extends Component {
     const animation = (
       <div className={styles.animation}>
         <Lottie
+          ariaRole={null}
```

Comment:
Just double checking - should this be null?

Reply:
yes that is intentional! lottie incorrectly sets roles and labels, so this is a way to prevent that in the interim. long term, we should start using the more up-to-date (forked) version of lottie which fixes this